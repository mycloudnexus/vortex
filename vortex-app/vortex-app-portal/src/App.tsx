import { FC } from 'react'
import { RouterProvider } from 'react-router-dom'
import { router } from './utils/routers'
import { ConfigProvider } from 'antd'
import { useAppStore } from './stores/app.store'

const App: FC = () => {
  const { mainColor } = useAppStore()
  return (
    <ConfigProvider
      theme={{
        token: {
          borderRadius: 0,
          colorPrimary: mainColor
        }
      }}
    >
      <RouterProvider router={router} />
    </ConfigProvider>
  )
}

export default App
