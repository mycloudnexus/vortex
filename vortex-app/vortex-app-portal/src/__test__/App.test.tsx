import { render } from '@/test/setupTest'
import App from '../App'

jest.mock('@auth0/auth0-react', () => ({
  Auth0Provider: jest.fn()
}))

describe('App test', () => {
  it('renders  app ', () => {
    const { container } = render(<App />)
    expect(container).toBeInTheDocument()
  })
  it('renders  with favicon ', () => {
    const { container } = render(<App />)
    expect(container.firstChild).toMatchSnapshot()
  })
})
