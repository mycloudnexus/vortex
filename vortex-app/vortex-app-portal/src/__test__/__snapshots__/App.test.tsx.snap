// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App test renders  with favicon  1`] = `null`;
